src/components/search/MobileFilterMenu.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes, faCaretDown } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@edx/paragon';
+
+import ClearCurrentRefinements from './ClearCurrentRefinements';
+
+const MobileFilterMenu = ({ children }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      {!isOpen && (
+        <Button
+          className="btn btn-block bg-white rounded-0 d-flex align-items-center justify-content-between"
+          onClick={() => setIsOpen(true)}
+        >
+          <span className="mr-2">Filters</span>
+          <FontAwesomeIcon icon={faCaretDown} />
+        </Button>
+      )}
+      <div
```

Comment:
would like to use Paragon's modal here but it's not quite flexible enough to support this design